packages/cli/src/commands/redis/maxmemory.ts

```diff
@@ -0,0 +1,41 @@
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+import redisApi from '../../lib/redis/api'
+
+export default class MaxMemory extends Command {
+  static topic = 'redis'
+  static description = 'set the key eviction policy'
+  static help = `Set the key eviction policy when instance reaches its storage limit. Available policies for key eviction include:
+
+  noeviction      # returns errors when memory limit is reached
+  allkeys-lfu     # removes less frequently used keys first
+  volatile-lfu    # removes less frequently used keys first that have an expiry set
+  allkeys-lru     # removes less recently used keys first
+  volatile-lru    # removes less recently used keys first that have an expiry set
+  allkeys-random  # evicts random keys
+  volatile-random # evicts random keys but only those that have an expiry set
+  volatile-ttl    # only evicts keys with an expiry set and a short TTL
+`
+  static aliases = ['redis']
+  static flags = {
+    app: flags.app({required: true}),
+    policy: flags.string({char: 'p', description: 'set policy name', required: true}),
+    remote: flags.remote(),
+  }
+
+  static args = {
+    database: Args.string(),
+  }
+
+  public async run(): Promise<void> {
+    const {flags, args} = await this.parse(MaxMemory)
+    const {app, policy} = flags
+    const {database} = args
+
+    const addon = await redisApi(app, database, false, this.heroku).getRedisAddon()
+    const {body: config}: {body: any} = await redisApi(app, database, false, this.heroku).request(`/redis/v0/databases/${addon.name}/config`, 'PATCH', {maxmemory_policy: policy})
```

Comment:
Ah nice, yeah will add this.

Reply:
ugh, this is a bit more complicated because of the use of an internally defined `request` function in the redis `api` utility.. hmm, maybe can just use heroku.request directly for this particular request? or maybe i can add another function to the api to do the patching as well..

Reply:
oh, i figured it out. just needed to add one of those <T> situations

Reply:
Yeah, I had that resolved on my other Redis PR. But I forgot to mention it.